src/_content/posts/call-for-proposals.md

```diff
@@ -0,0 +1,27 @@
+---
+author: communications Team
+category: General
+published_datetime: 2025-03-04 06:00:00
+cover:
+    url: /assets/img/2025/54089577525_de6f9bcb72_w.jpg
+    alt: "Sheena O'Connel speaking at DjangoCon US 2024"
+title: "The Call for Proposals for DjangoCon US 2025 Is Now Open!"
+---
+
+We are excited that DjangoCon US 2025 is now open for talk submissions: [call for proposals](https://pretalx.com/djangocon-us-2025/cfp)! The deadline for submissions is April 27th, 2025 at [Noon CST](https://time.is/1200PM_27_April_2025_in_Chicago?DjangoCon_US_CFP_Closes).
+
+<!-- excerpt -->
+
+We invite you to submit your proposal no matter your background or experience level with Django. Proposals can be from a wide range of topics; non-Django and community topics are welcome. You can look at our [talk lineup](https://2024.djangocon.us/news/announcing-lineup/) from last year for reference.
+
+We fancy first-timers! If you haven’t spoken at a conference or given a tutorial before, this is your invitation to do so. Don’t let the idea that you’re not famous or an expert stop you from submitting. It certainly won’t stop us from selecting your talk or tutorial and it won’t stop the audience from enjoying it! If you still need help decding on a topic take a look at our own Drew Wentzel's [blog post](https://winstel.dev/2025/02/12/talk-ideas-for-dcus-25/) for some ideas.
```

Comment:
```suggestion
We fancy first-timers! If you haven’t spoken at a conference or given a tutorial before, this is your invitation to do so. Don’t let the idea that you’re not famous or an expert stop you from submitting. It certainly won’t stop us from selecting your talk or tutorial and it won’t stop the audience from enjoying it! If you still need help decding on a topic take a look at our own Drew Winstel's [blog post](https://winstel.dev/2025/02/12/talk-ideas-for-dcus-25/) for some ideas.
```

That's a new way to misspell my name (and I've seen a lot)! 😆

Reply:
How in the world did I do that!? LOL.